ts/input/tex/textcomp/TextcompMappings.ts

```diff
@@ -0,0 +1,191 @@
+import ParseUtil from '../ParseUtil.js';
+import {TextParser} from '../textmacros/TextParser.js';
+
+
+/**
+ * Identifiers from the Textcomp package.
+ */
+new CommandMap('textcomp-macros', {
+
+  // Table 3: Predefined LATEX 2ε Text-Mode Commands
+  'textasciicircum':     ['Insert', '\u005E'],
+  'textasciitilde':      ['Insert', '\u007E'],
+  'textasteriskcentered': ['Insert', '\u002A'],
+  'textbackslash':       ['Insert', '\u005C'],
+  'textbar':             ['Insert', '\u007C'],
+  'textbraceleft':       ['Insert', '\u007B'],
+  'textbraceright':      ['Insert', '\u007D'],
+  'textbullet':          ['Insert', '\u2022'],
+  'textdagger':          ['Insert', '\u2020'],
+  'textdaggerdbl':       ['Insert', '\u2021'],
+  'textellipsis':        ['Insert', '\u2026'],
+  'textemdash':          ['Insert', '\u2014'],
+  'textendash':          ['Insert', '\u2013'],
+  'textexclamdown':      ['Insert', '\u00A1'],
+  'textgreater':         ['Insert', '\u003E'],
+  'textless':            ['Insert', '\u003C'],
+  'textordfeminine':     ['Insert', '\u00AA'],
+  'textordmasculine':    ['Insert', '\u00BA'],
+  'textparagraph':       ['Insert', '\u00B6'],
+  'textperiodcentered':  ['Insert', '\u00B7'],
+  'textquestiondown':    ['Insert', '\u00BF'],
+  'textquotedblleft':    ['Insert', '\u201C'],
+  'textquotedblright':   ['Insert', '\u201D'],
+  'textquoteleft':       ['Insert', '\u2018'],
+  'textquoteright':      ['Insert', '\u2019'],
+  'textsection':         ['Insert', '\u00A7'],
+  'textunderscore':      ['Insert', '\u005F'],
+  'textvisiblespace':    ['Insert', '\u2423'],
+
+  // Table 12: textcomp Diacritics
+  'textacutedbl':        ['Insert', '\u02DD'],
+  'textasciiacute':      ['Insert', '\u00B4'],
+  'textasciibreve':      ['Insert', '\u02D8'],
+  'textasciicaron':      ['Insert', '\u02C7'],
+  'textasciidieresis':   ['Insert', '\u00A8'],
+  'textasciimacron':     ['Insert', '\u00AF'],
+  'textgravedbl':        ['Insert', '\u02F5'],
+  'texttildelow':        ['Insert', '\u02F7'],
+
+  // Table 13: textcomp Currency Symbols
+  'textbaht':            ['Insert', '\u0E3F'],
+  'textcent':            ['Insert', '\u00A2'],
+  'textcolonmonetary':   ['Insert', '\u20A1'],
+  'textcurrency':        ['Insert', '\u00A4'],
+  'textdollar':          ['Insert', '\u0024'],
+  'textdong':            ['Insert', '\u20AB'],
+  'texteuro':            ['Insert', '\u20AC'],
+  'textflorin':          ['Insert', '\u0192'],
+  'textguarani':         ['Insert', '\u20B2'],
+  'textlira':            ['Insert', '\u20A4'],
+  'textnaira':           ['Insert', '\u20A6'],
+  'textpeso':            ['Insert', '\u20B1'],
+  'textsterling':        ['Insert', '\u00A3'],
+  'textwon':             ['Insert', '\u20A9'],
+  'textyen':             ['Insert', '\u00A5'],
+
+  // Table 15: textcomp Legal Symbols
+  'textcircledP':        ['Insert', '\u2117'],
+  'textcompwordmark':    ['Insert', '\u200C'],
+  'textcopyleft':        ['Insert', '\u1F12F'],
```

Comment:
This needs to be `\u{1F12F}` otherwise it is U+1F12 followed by an F.

Reply:
Thanks for catching this. Of course it still produces an error in chtml until PR #723 is merged.